### ISSUE TITLE
Cut a release?

### ISSUE THREAD

**Issue by kozross** [ekmett/semigroupoids#115]
There's been considerable change, and it'd be nice to have 5.4 on Hackage.

@RyanGlScott - is there anything we want fixed or merged before this?

**Comment by RyanGlScott** [comment]
I'm fine with making a new release. One small correction—it would be version 5.3.6, not 5.4, after #114.

Do you want to push through https://github.com/ekmett/semigroupoids/pull/103#issuecomment-932662163 before the next release?

**Comment by kozross** [comment]
Wait, I'm a bit confused. The last few merges have been changelogged under 5.4 - would this have to change? I'm happy to get #103 over the line.

**Comment by RyanGlScott** [comment]
> The last few merges have been changelogged under 5.4 - would this have to change?

No need, I adjusted the changelog/`@since` annotations to `5.3.6` in 9fbea3d53d0f0079ab708f8520b724d67adbca60 (part of #114).

> I'm happy to get #103 over the line.

Sounds good to me. In that case, I'll hold off on making a new release pending that change. Thanks!
